1.data_preprocessing/1.1.make_pseudobulks.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>Since you have the subtype calls, it might be useful to also include a figure  of the single-cell subtype calls by samples  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NDavidsonLab/BuDDI_HGSC_schildkraut_analysis/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>